tests/api/test_sources_endpoint.py

```diff
@@ -0,0 +1,141 @@
+import json
+import pytest
+
+from share.models import Source, ShareUser
+
+
+PROPER_ICON_URL = 'https://staging-cdn.osf.io/preprints-assets/bitss/square_color_no_transparent.png'
```

Comment:
Ended up using HTTPretty even though they don't officially support python3 because mock-requests -- to my knowledge -- doesn't provide a way to mock requests that are in the codebase rather than the test file.